packages/grafana-ui/src/components/Forms/InlineField.tsx

```diff
@@ -23,7 +23,7 @@ export interface Props extends Omit<FieldProps, 'css' | 'horizontal' | 'descript
   /** Make field's background transparent */
   transparent?: boolean;
   /** Error message to display */
-  error?: string | null;
+  error?: ReactNode;
```

Comment:
Is `null` a subset of ReactNode?

Reply:
field being optional _and_ `| null` is annoying. I'm for this cleanup.

Reply:
From React's index.d.ts: 
```
    type ReactNode =
        | ReactElement
        | string
        | number
        | Iterable<ReactNode>
        | ReactPortal
        | boolean
        | null
        | undefined
```